substrate-archive/src/database/queries.rs

```diff
@@ -64,6 +64,7 @@ pub(crate) async fn missing_blocks_min_max(
 ) -> Result<HashSet<u32>> {
     let min = i32::try_from(min).unwrap_or(i32::MAX);
     let max_block_load = i64::try_from(max_block_load).unwrap_or(i64::MAX);
+    #[allow(clippy::toplevel_ref_arg)]
```

Comment:
this is caused by `sqlx::query_as!` and not by us, correct?

It just a [style warning](https://rust-lang.github.io/rust-clippy/master/index.html) but perhaps open an issue upstream?!

Reply:
There already is an issue up for it https://github.com/launchbadge/sqlx/issues/594